stdlib/3/hmac.pyi

```diff
@@ -0,0 +1,27 @@
+# Stubs for hmac (Python 3.5)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from typing import Any
+from _operator import _compare_digest as compare_digest
```

Comment:
If you rebase the tests should work -- I just made the tests "pass" by applying some surgery to the sqlalchemy stubs (they need more work, but I'm not a sqlalchemy user).

Is it possible that the stub was never loaded in your own usage? (Easy to test by introducing a deliberate syntax error.) There actually us a stub for operator already (in typeshed/stdlib/2and3).